data/data.json

```diff
@@ -4414,6 +4414,10 @@
    "maxRKEVersion": "1.0.0",
    "maxRancherVersion": "2.3.3"
   },
+  "v1.15.11-rancher1-1": {
+   "minRKEVersion": "1.0.0",
+   "minRancherVersion": "2.3.3"
+  },
   "v1.15.11-rancher1-2": {
```

Comment:
Is there any reason why we can't just update this instead of adding something new?

Reply:
This isn't a min version but needs each version specifically.